client-src/elements/chromedash-userlist.ts

```diff
@@ -75,14 +71,17 @@ class ChromedashUserlist extends LitElement {
   }
 
   _onAdminToggle() {
-    const formEl = this.shadowRoot.querySelector('form');
-    const adminCheckbox = formEl.querySelector('input[name="is_admin"]');
-    const siteEditorCheckbox = formEl.querySelector(
+    const formEl: HTMLFormElement | null =
+      this.renderRoot.querySelector('form');
+    const adminCheckbox = formEl?.querySelector(
+      'input[name="is_admin"]'
+    ) as SlCheckbox;
+    const siteEditorCheckbox = formEl?.querySelector(
       'input[name="is_site_editor"]'
-    );
+    ) as SlCheckbox;
     // Admins will always be site editors, so if the admin box is checked,
     // the site editor box is also checked and disabled.
-    if (adminCheckbox.checked) {
+    if (adminCheckbox?.checked) {
```

Comment:
I'd use `querySelector(...)!` instead of `?.checked` to deal with the nullability here, because the target element is unconditionally in the rendered HTML.